substrate/frame/aura/src/lib.rs

```diff
@@ -42,17 +42,20 @@ extern crate alloc;
 
 use alloc::vec::Vec;
 use codec::{Decode, Encode, MaxEncodedLen};
-use frame_support::{
-	traits::{DisabledValidators, FindAuthor, Get, OnTimestampSet, OneSessionHandler},
-	BoundedSlice, BoundedVec, ConsensusEngineId, Parameter,
+use frame::{
+	deps::{
+		frame_support::{ConsensusEngineId, Parameter},
```

Comment:
Please read the new checklist named in the issue, and update the PR to adhere to it. Thank you!